pkg/operator/operator_test.go

```diff
@@ -131,14 +132,15 @@ func TestOperatorSync_NoOp(t *testing.T) {
 			stopCh := make(<-chan struct{})
 			optr := newFakeOperator(nil, []runtime.Object{infra}, stopCh)
 			optr.queue.Add("trigger")
-			go optr.Run(2, stopCh)
+			go optr.Run(1, stopCh)
 
-			err := wait.PollImmediate(1*time.Second, 5*time.Second, func() (bool, error) {
+			err := wait.PollImmediate(5*time.Second, 1*time.Second, func() (bool, error) {
```

Comment:
This will only poll once and timeout with these values swapped, is that the desired behaviour?

Reply:
typo, let me fix it.